src/types/kubectl.ts

```diff
@@ -56,16 +56,24 @@ export class Kubectl {
    public async describe(
       resourceType: string,
       resourceName: string,
-      silent: boolean = false
+      silent: boolean = false,
+      namespace?: string
```

Comment:
💡thought to share:  Nullish with `?` but type is passed as `string` not `string | null`

Reply:
I was curious about that too. Do you know what the best practice is?

I opted for just `string` instead of `string | undefined` because that's what they do in the [typescript official docs](https://www.typescriptlang.org/docs/handbook/2/functions.html).

Reply:
☕️🥷 indeed! ++ , thinking out loud - For anything unintuitive it’s better to mention type upfront. Totally with you, thinking about the function contract for types.

Given you have thought this far, I will go with whatever decision we choose and we have it archived here anyways.